doc/index.rst

```diff
@@ -175,6 +175,13 @@ Note that some of these currently tied only to a single option, but we still lea
 * **CLANG_FORMAT_CHECK** (default: not set. Value range: [``<format-style>``|``file``]): If set, run the `clang-format <https://clang.llvm.org/docs/ClangFormat.html>`__ check. Set the argument to ``file`` if the style configuration should be loaded from a ``.clang-format`` file, located in one of the parent directories of the source file.
 * **CLANG_FORMAT_VERSION** (default: not set): Version of clang-format to install and use (relates to both the apt package name as well as the executable), e.g., ``CLANG_FORMAT_VERSION=3.8``.
 * **CLANG_TIDY** (default: not set. Value range: [``true``|``pedantic``]): If set, run `clang.tidy <https://clang.llvm.org/extra/clang-tidy/>`__ to check the code in all packages and fail in case of errors. If ``pedantic``, warnings will be treated as errors as well.
+* **CLANG_TIDY_BASE_REF** (default: not set.): If set, clang-tidy tests will be performed on files only that changed since the given ref. If not set, clang-tidy checks are performed on all files.
```

Comment:
nitpick: please put it if after `CLANG_TIDY_ARGS` (here and in the next two chunks)

Reply:
I'm not quite sure what you meant by "the next two chunks". But I guess you want to have the variables sorted alphabetically.
Done.